Runtime/Scripts/Extensions/MaterialExtensions.cs

```diff
@@ -19,6 +19,11 @@ internal static class MaterialExtensions {
             existingRenderTexture.Release();
 
         mat.SetTexture(nameID, texture);
+            
+        // Ensure main texture is set if the name is not _MainTex!
```

Comment:
I don't follow. The code underneath ensures that main texture is set **if** the name is _MainText. Am I missing something?